index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ramiro Aquistapace</title>
+    <link rel="stylesheet" href="style.css" />
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap");
+    </style>
+</head>
+<body>
+    <div class="wrapper">
+      <header class="header">
+        <h3 class="mylogo">My Logo</h3>
+        <img alt="" src="Union.png" class="icon" />
+      </header>
+      <div class="container">
+        <h1>I’m Ramiro Glad to see you!</h1>
+        <p>
+          I’m a software developer and a incredible Micronaut! I can help you
+          build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.
+        </p>
```

Comment:
- [ ] Please remove the boldness of this text to match the Figma design:
![image](https://user-images.githubusercontent.com/38522578/129119451-ec9827a8-adf6-421c-863f-d768ed6b45ef.png)